recommender/recommender.py

```diff
@@ -976,16 +975,16 @@ def add_xml_to_node(self, node):
         """
         node.tag = 'recommender'
 
-        node.set('intro_enabled', str(self.intro_enabled))
-        node.set('DISABLE_DEV_UX', str(self.client_side_settings['DISABLE_DEV_UX']))
-        node.set('ENTRIES_PER_PAGE', str(self.client_side_settings['ENTRIES_PER_PAGE']))
-        node.set('PAGE_SPAN', str(self.client_side_settings['PAGE_SPAN']))
+        node.set('intro_enabled', 'true' if (self.intro_enabled) else 'false')
+        node.set('disable_dev_ux', 'true' if (self.client_configuration['disable_dev_ux']) else 'false')
+        node.set('entries_per_page', str(self.client_configuration['entries_per_page']))
+        node.set('page_span', str(self.client_configuration['page_span']))
 
         el = etree.SubElement(node, 'resources')
         ## Note: The line below does not work in edX platform. 
         ## We should figure out if the appropriate scope is available during import/export
         ## TODO: Talk to Cale
```

Comment:
This is not something I expect we can fix quickly, but we should talk to @cpennington. 

Cale: Student are contributing content to our courses. When we try to export, those fields are blank. Am I correct in assuming that this scope is not available during export? If so, is this something that could be fixed quickly and easily?